shared.go

```diff
@@ -495,6 +498,8 @@ func curveName(crv elliptic.Curve) (string, error) {
 		return "P-384", nil
 	case elliptic.P521():
 		return "P-521", nil
+	case btcec.S256():
+		return "secp256k1", nil
```

Comment:
Add comment here, like `// sic!` maybe. Similar to what we do for the 512/521 bits.